setup.py

```diff
@@ -73,6 +73,7 @@ def check_provided(distribution, min_version, max_version=None, optional=False):
     author_email='cgl-toil@googlegroups.com',
     url="https://github.com/BD2KGenomics/toil-scripts",
     install_requires=[
+        'toil-lib==1.0.2',
         'tqdm==3.8.0',  # FIXME: Remove once ADAM stops using it (superfluous import)
```

Comment:
Small aside, but I think we can get rid of the `tqdm` requirement. I don't see it used anywhere in the codebase.